composer.json

```diff
@@ -22,7 +22,8 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0"
+        "php": "^7.3 || ^8.0",
+        "symfony/process": "^5"
```

Comment:
Why not version six?

Reply:
Because version 6 requires php version 8 if I'm not mistaken.

Reply:
"php": ">=7.2.5",

Reply:
You are right

Reply:
You are right